<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreditWindow</class>
 <widget class="QDialog" name="CreditWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>310</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>310</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>310</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_overpayment">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>50</y>
     <width>150</width>
     <height>21</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_take_credit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>19</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Общая сумма кредита</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_obshaya">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>81</y>
     <width>137</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Общая выплата:</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>154</y>
     <width>661</width>
     <height>141</height>
    </rect>
   </property>
   <property name="showDropIndicator" stdset="0">
    <bool>false</bool>
   </property>
   <property name="showGrid">
    <bool>false</bool>
   </property>
   <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderShowSortIndicator" stdset="0">
    <bool>true</bool>
   </attribute>
  </widget>
  <widget class="QLabel" name="label_sym_percent">
   <property name="geometry">
    <rect>
     <x>294</x>
     <y>81</y>
     <width>45</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_egemesyachi">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>19</y>
     <width>137</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>120</width>
     <height>0</height>
    </size>
   </property>
   <property name="text">
    <string>Ежемесячный платёж:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_m_payment">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>19</y>
     <width>150</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>150</width>
     <height>0</height>
    </size>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_rub">
   <property name="geometry">
    <rect>
     <x>294</x>
     <y>19</y>
     <width>45</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>рублей</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_all_payment">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>81</y>
     <width>150</width>
     <height>21</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_percent">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>81</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Процентная ставка</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_pereplata">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>50</y>
     <width>137</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Переплата по кедиту:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_srok_credita">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Срок кредита</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="DSB_credit_amount">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>20</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <double>1000.000000000000000</double>
   </property>
   <property name="maximum">
    <double>1000000000.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>1000.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="DSB_credit_percent">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>80</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <double>100.000000000000000</double>
   </property>
   <property name="value">
    <double>0.010000000000000</double>
   </property>
  </widget>
  <widget class="QSpinBox" name="SB_period">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>50</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>60</number>
   </property>
   <property name="value">
    <number>12</number>
   </property>
  </widget>
  <widget class="QComboBox" name="CB_period">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>46</y>
     <width>101</width>
     <height>32</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Месяцев</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Лет</string>
    </property>
   </item>
  </widget>
  <widget class="QRadioButton" name="radioButton_ann">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>120</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Аннуитетный</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_calculate">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>115</y>
     <width>199</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Рассчитать</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_diff">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>120</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>165</width>
     <height>0</height>
    </size>
   </property>
   <property name="text">
    <string>Дифференцированный</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
